src/styles/blocks/reviews.scss

```diff
@@ -0,0 +1,11 @@
+.reviews {
+  display: flex;
+  margin-left: 37px;
+  margin-right: 0;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 10px;
+  text-align: right;
+  color: #060b35;
```

Comment:
The `font-size` of `10px` is quite small for readability. Make sure that this size is in line with the design requirements. If it is not a specific requirement, consider using a size that is more accessible for users, such as `12px` or larger.